cloudify_agent/tests/api/pm/__init__.py

```diff
@@ -410,6 +410,19 @@ def _check_env_var(var, expected_value):
         for key, value in expected.iteritems():
             _check_env_var(key, value)
 
+        def _check_env_path():
+            _path = self.celery.send_task(
+                name='mock_plugin.tasks.get_env_variable',
+                queue=daemon.queue,
+                args=['PATH']).get(timeout=5)
+            if os.name == 'nt':
+                self.nose.tools.assert_in(
+                    str(daemon.virtualenv_path) + '\scripts', _path)
```

Comment:
instead of `str(daemon.virtualenv_path) + '\scripts'`
please do '{0}\scripts`.format(daemon.virtualenv_path)
btw: shouldn't it be `\Scripts` with a capital s?